pkg/issuer/venafi/client/venaficlient.go

```diff
@@ -91,10 +91,10 @@ func New(namespace string, secretsLister corelisters.SecretLister, issuer cmapi.
 // configForIssuer will convert a cert-manager Venafi issuer into a vcert.Config
 // that can be used to instantiate an API client.
 func configForIssuer(iss cmapi.GenericIssuer, secretsLister corelisters.SecretLister, namespace string) (*vcert.Config, error) {
-	venCfg := iss.GetSpec().Venafi
+	venaCfg := iss.GetSpec().Venafi
```

Comment:
I'm not confident about this change, but given that the thing is called `Venafi`, it seems like `vena` is clearer than `ven`.

Reply:
I am fine with this change. @wallrj thoughts?